isisd/isis_te.c

```diff
@@ -450,30 +499,35 @@ DEFUN (show_isis_mpls_te_interface,
        "Interface information\n"
        "Interface name\n")
 {
-	struct listnode *anode, *cnode;
+	struct listnode *anode, *cnode, *nnode, *inode;
 	struct isis_area *area;
 	struct isis_circuit *circuit;
 	struct interface *ifp;
 	int idx_interface = 4;
+	struct isis *isis = NULL;
 
-	if (!isis) {
+	if (!im) {
 		vty_out(vty, "IS-IS Routing Process not enabled\n");
 		return CMD_SUCCESS;
 	}
 
 	if (argc == idx_interface) {
 		/* Show All Interfaces. */
-		for (ALL_LIST_ELEMENTS_RO(isis->area_list, anode, area)) {
+		for (ALL_LIST_ELEMENTS(im->isis, nnode, inode, isis)) {
```

Comment:
This is not a vrf all command.  It must be fixed.

Reply:
Okay, will fix it.  
Actually by mistake this command  " show isis vrf <vrf-name> mpls-te interface " got missed.